tfe/tool_helpers.go

```diff
@@ -0,0 +1,42 @@
+package tfe
+
+import (
+	"fmt"
+
+	tfe "github.com/hashicorp/go-tfe"
+)
+
+// fetchTerraformVersionID returns a Terraform Version ID for the given Terraform version number
+func fetchTerraformVersionID(version string, client *tfe.Client) (string, error) {
```

Comment:
Is it an error when the specified version is not found? I'm thinking it should be but it's not reflected in the code. (The caller does not check for empty id)